site/src/components/Footer.scss

```diff
@@ -1,33 +1,31 @@
 @import "~@kaizen/design-tokens/sass/color";
-@import "~@kaizen/deprecated-component-library-helpers/styles/type";
-@import "~@kaizen/deprecated-component-library-helpers/styles/color";
+@import "~@kaizen/design-tokens/sass/spacing";
 
-$reverse-variant-color: $color-purple-700;
+$reverse-variant-color: $color-blue-100;
 
 .footerExtraContent {
   .reverseVariant & {
     background-color: $reverse-variant-color;
-    color: white;
   }
 }
 
 .footer {
   display: flex;
   justify-content: center;
   align-items: center;
-  height: $ca-grid * 3;
+  height: $spacing-xxxl;
 
   .reverseVariant & {
     background-color: $reverse-variant-color;
-    color: white;
+    color: $color-purple-800;
   }
 }
 
 .footerInner {
   display: flex;
   align-items: center;
   justify-content: space-evenly;
-  max-width: $ca-grid * 16;
+  max-width: calc(#{$spacing-md} * 16);
```

Comment:
Is there a reason why this is `$spacing-md` and the next file is `var(--spacing-md)`?